### PR TITLE
docs: update getting-started.md

### PR DESCRIPTION
**Pull request by smira**
Small fixes for getting started guide.

Signed-off-by: Andrey Smirnov <andrey.smirnov@talos-systems.com>

> See `make help` for a description of the available targets.

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/4989)

### CODE REVIEW

**Review by frezbo — APPROVED** [review]

**Review by andrewrynhard — APPROVED** [review]

File: website/content/docs/v0.15/Introduction/getting-started.md [hunk 1]
```diff
@@ -371,8 +350,8 @@ We would set those in the `talosconfig` with:
 
 The node is the target node on which you wish to perform the API call.
 
-Keep in mind, when specifying nodes that their IPs and/or hostnames are as seen by the endpoint servers, not as from the client.
-This is because all connections are proxied first through the endpoints.
+Keep in mind, when specifying nodes that their IPs and/or hostnames are *as seen by the endpoint servers*, not as from the client.
```
Comment (Ulexus) [inline website/content/docs/v0.15/Introduction/getting-started.md:353]: "Keep in mind, when specifying nodes_,_ that their ..." (missing comma at the end of the clause).

**Review by Ulexus — COMMENTED** [review]
A few minor rewording suggestions

**Comment by smira** [comment]
@steverfrancis please review Seán's comments, as it's not my changes anyways :)

File: website/content/docs/v0.15/Introduction/getting-started.md [hunk 2]
```diff
@@ -21,9 +20,9 @@ In general you will need to:
 
 ### `talosctl`
 
-The `talosctl` tool provides a CLI tool which interfaces with the Talos API in
+ `talosctl` provides a CLI tool which interfaces with the Talos API in
```
[the remark this answers is not in the document]
Comment (steverfrancis) [reply website/content/docs/v0.15/Introduction/getting-started.md:23]: Agreed

File: website/content/docs/v0.15/Introduction/getting-started.md [hunk 3]
```diff
@@ -21,9 +20,9 @@ In general you will need to:
...
 an easy manner.
-It also includes a number of useful tools for creating and managing your clusters.
+It also includes a number of useful tools for creating and managing clusters.
```
[the remark this answers is not in the document]
Comment (steverfrancis) [reply website/content/docs/v0.15/Introduction/getting-started.md:25]: Done

File: website/content/docs/v0.15/Introduction/getting-started.md [hunk 4]
```diff
@@ -52,19 +51,18 @@ The latest ISO image can be found on the Github [Releases](https://github.com/ta
...
-
 When booted from the ISO, Talos will run in RAM, and it will not install itself
 until it is provided a configuration.
 Thus, it is safe to boot the ISO onto any machine.
 
 ### Alternative Booting
 
-If you wish to use a different boot mechanism (such as network boot or a custom ISO), there
-are a number of required kernel parameters.
+For self-built media and network booting, you can use the kernel and initramfs:
```
[the remark this answers is not in the document]
Comment (steverfrancis) [reply website/content/docs/v0.15/Introduction/getting-started.md:60]: yup

**Review by Ulexus — APPROVED** [review]

**Comment by smira** [comment]
squashed commits, fixed linting errors

**Pull request merged**